docs/deployment/config_descriptions.md

```diff
@@ -46,6 +46,26 @@ A representation of the stack name that has dashes removed. This will be used fo
 
 Configure your virtual private cloud.  You can find `<vpc-id>` and `<subnet-id>` values on the [VPC Dashboard](https://console.aws.amazon.com/vpc/home?region=us-east-1#). `vpcId` from [Your VPCs](https://console.aws.amazon.com/vpc/home?region=us-east-1#vpcs:), and `subnets` [here](https://console.aws.amazon.com/vpc/home?region=us-east-1#subnets:). When you choose a subnet, be sure to also note its availability zone, to configure `ecs`.
 
+## cmr
+
+Configuration required for Cumulus integration with CMR services. The most obvious example of this integration is the `PostToCmr` Cumulus [task](https://github.com/nasa/cumulus/tree/master/tasks/post-to-cmr). `username` and `password` should be a CMR service username/password pair. Often in our documentation you will see these set as:
+```yaml
+cmr:
+  username: '{{CMR_USERNAME}}'
+  password: '{{CMR_PASSWORD}}'
+```
+`{{CMR_USERNAME}}` references a `CMR_USERNAME` value set in the `app/.env` file. `{{CMR_PASSWORD}}` references `CMR_PASSWORD` in the same way.
```

Comment:
I feel like it might be clearer to show and give an example as in these docs: https://nasa.github.io/cumulus/docs/next/data-cookbooks/browse-generation#cmr

As an aside, it seems like maybe that documentation page could point to this one for info on configuring CMR? Reducing duplication and how many places in the docs we have to maintain this info

Reply:
That's a good idea.

Reply:
https://github.com/nasa/cumulus/pull/937/commits/5db4bf84d4abf19cdb5249aa6b0c11a263374950